### PR TITLE
Update timeseries_classification_transformer tutorial with good accuracy

### PR DESCRIPTION
**Pull request by mehtamansi29**
timeseries_classification_transformer model creation data_format="channels_last" is giving less model parameters and degrades accuracy. Update example with data_format="channels_first". Fixes [#20627](https://github.com/keras-team/keras/issues/20627)

### CODE REVIEW

File: examples/timeseries/timeseries_classification_transformer.py [hunk 1]
```diff
@@ -111,8 +121,9 @@ def build_model(
     x = inputs
     for _ in range(num_transformer_blocks):
         x = transformer_encoder(x, head_size, num_heads, ff_dim, dropout)
-
-    x = layers.GlobalAveragePooling1D(data_format="channels_last")(x)
+    print(f"Transformer Encoder: {x}")
+    x = layers.GlobalAveragePooling1D(data_format="channels_first")(x)
```
[the remark this answers is not in the document]
Comment (mehtamansi29) [reply examples/timeseries/timeseries_classification_transformer.py:125]: @fchollet - As per code example earlier data_format="channels_last" reduces the model parameter and accuracy degradation on GPU in latest keras. data_format="channels_first" in pooling also reduce the number of time steps in same GPU environment and get good accuracy.
